components/event/EventCard.js

```diff
@@ -18,7 +18,7 @@ export default function EventCard({ event, username }) {
 
   return (
     <li
-      className="py-4 border-l-3 mb-4 pl-2 rounded-lg shadow-lg transition duration-350 hover:scale-105 duration-500 ease-in-out"
+      className="py-4 border-l-3 mb-4 pl-2 rounded-lg shadow-lg transition duration-350 dark:bg-[#233042] hover:scale-105 duration-500 ease-in-out"
```

Comment:
`bg-[#233042]` is same as `gray-800` can we use the name instead of hash?

Reply:
The hash is actually a bit brighter than gray-800 I tried using gray-800 on the card. Looks a bit weird